CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+These are general guidelines for contributing for all WHATWG standards.
+
+### Pull requests
+
+Leave the **Allow edits from maintainers** option enabled to allow reviewers to fix trivial issues directly on your branch rather than needing to write review comments asking you make the edits.  For more details, see [Improving collaboration with forks](https://github.com/blog/2247-improving-collaboration-with-forks) in the GitHub Blog.
+
+### Tests
+
+For normative changes, a corresponding [web-platform-tests](https://github.com/w3c/web-platform-tests) PR is highly appreciated. The author and reviewer can be different from the spec PR. If current behavior is unclear, writing tests first can help inform the discussion. Typically, both PRs will be merged at the same time.
```

Comment:
@annevk, here I had to change the wording from "the HTML Standard PR" to "the spec PR" and I remember you didn't want this wording originally. Thoughts?